py/desispec/scripts/stdstars.py

```diff
@@ -468,19 +493,21 @@ def main(args) :
         model_filters["GAIA-" + band] = load_gaia_filter(band=band, dr=2)
 
     log.info("computing model mags for %s"%sorted(model_filters.keys()))
```

Comment:
please move this log.info into the "if rank == 0" block so that the log messages don't incorrectly imply that all ranks are computing model mags.